core/src/test/java/com/capitalone/dashboard/MarkdownTest.java

```diff
@@ -60,9 +61,18 @@ private File getProjectRoot() {
     }
 
     CharSequence fromFile(String filename) throws IOException {
-        try (FileChannel channel = new FileInputStream(filename).getChannel()) {
+    	FileInputStream input = null;
+        try {
+        	input = new FileInputStream(filename);
+        	FileChannel channel = input.getChannel();
             ByteBuffer buffer = channel.map(FileChannel.MapMode.READ_ONLY, 0, (int) channel.size());
-            return Charset.forName("8859_1").newDecoder().decode(buffer);
+            CharBuffer result = Charset.forName("8859_1").newDecoder().decode(buffer);
+            input.close();
+			return result;
+        } catch (Exception e) {
```

Comment:
See #350  - Eclipse identified we where not close the FileChannel resource returned, so my goal was to fix that.

This was the original code:

    CharSequence fromFile(String filename) throws IOException {
        try (FileChannel channel = new FileInputStream(filename).getChannel()) {
            ByteBuffer buffer = channel.map(FileChannel.MapMode.READ_ONLY, 0, (int) channel.size());
            return Charset.forName("8859_1").newDecoder().decode(buffer);
        }
    }